ui/src/album/styles.js

```diff
@@ -26,6 +33,9 @@ export const useStyles = makeStyles((theme) => ({
       height: '15em',
       width: '15em',
     },
+    '&:hover $playButton': {
```

Comment:
`<PlayButton>` must be always visible in AlbumDetail on mobile, or maybe remove the component when on mobile, as we already have a play button just bellow the cover?